protos/apparmor-prompting.proto

```diff
@@ -87,6 +88,7 @@ enum HomePermission {
 message GetCurrentPromptResponse {
     oneof prompt {
         HomePrompt home_prompt = 1;
+        CameraPrompt camera_prompt = 2;
```

Comment:
here, ofc, we don’t change the order between Home and Camera for API compatibility, but it doesn’t mean in the code above that we forcibly have to follow the same order.